storage/filesystem/internal/dotgit/dotgit.go

```diff
@@ -375,7 +375,7 @@ func (d *DotGit) findPackedRefsInFile(f billy.File) ([]*plumbing.Reference, erro
 	return refs, s.Err()
 }
 
-func (d *DotGit) findPackedRefs() ([]*plumbing.Reference, error) {
+func (d *DotGit) findPackedRefs() (r []*plumbing.Reference, error error) {
```

Comment:
this one is called `error` instead of `err`, and you are using err on the code. Is a leftover or am I missing something?

Reply:
It's my sloppy fingers, thanks! Just fixed it.